generators/entity-client/files.js

```diff
@@ -379,11 +379,29 @@ const vueFiles = {
     ],
 };
 
+const cypressFiles = {
+    testsCypress: [
+        {
+            condition: generator => generator.cypressTests && !generator.embedded,
+            path: CLIENT_TEST_SRC_DIR,
+            templates: [
+                'cypress/support/entity.ts',
+                'cypress/support/index.ts',
```

Comment:
Move both files to cypress generator, otherwise it will be overwritten for each entity.